lib/cookie.js

```diff
@@ -36,11 +36,11 @@ module.exports = class Cookie {
 }
 
 function getExpires (cookieOpts) {
-  let expires = null
+  if (cookieOpts.maxAge) {
+    return new Date(Date.now() + cookieOpts.maxAge)
+  }
   if (cookieOpts.expires) {
-    expires = cookieOpts.expires
-  } else if (cookieOpts.maxAge) {
-    expires = new Date(Date.now() + cookieOpts.maxAge)
+    return cookieOpts.expires
```

Comment:
maxAge has priority over expires, according to the docs.

However, because Session was calling `touch` in its constructor, it was still doing the right behavior.

I removed that `touch()` because it is unnecessary, so I had to fix this